spec/spec_helper.rb

```diff
@@ -243,6 +243,9 @@ def self.from(system_exit)
 
     Chef::Log.setup!
 
+    Chef::Config[:log_level] = :fatal
+    Chef::Log.level(Chef::Config[:log_level])
```

Comment:
`Chef::Config[:log_level]` gets reset on every `:each` down below this doesn't it?

Reply:
I added these two lines just below where `Chef::Config` and `Chef::Log` are reset. I'm fairly sure `Chef::Config[:log_level]` is being set because of the changes required to fix the test failures in `spec/unit/application_spec.rb` that started in this PR. https://github.com/chef/chef/pull/10188/files#diff-874b8ae1db0389e90526e97d2d53695b

Reply:
ah right, i thought Chef::Log.setup! was outside the tight loop for some reason...